transform-plugins/src/main/java/io/cdap/plugin/CloneRecord.java

```diff
@@ -59,6 +67,25 @@ public void prepareRun(StageSubmitterContext context) throws Exception {
     FailureCollector collector = context.getFailureCollector();
     config.validate(collector);
     collector.getOrThrowException();
+
+    Schema inputSchema = context.getInputSchema();
```

Comment:
This is incorrect, this transform just copies the record n times based on the config. So the input and output schema will not change, your field operations should be a list of one-to-one field mapping from one input field to one output field